File: app/src/androidTest/java/com/mbmc/template/helper/TestRunner.java
package com.mbmc.template.helper;

import android.app.Application;
import android.content.Context;
import android.support.test.runner.AndroidJUnitRunner;

import com.mbmc.template.TestApplication;


public class TestRunner extends AndroidJUnitRunner {

    @Override
    public Application newApplication(ClassLoader cl, String className, Context context)
            throws InstantiationException, IllegalAccessException, ClassNotFoundException {
        return super.newApplication(cl, TestApplication.class.getCanonicalName(), context);
    }

}

File: presentation/src/androidTest/kotlin/com/mbmc/template/helper/TestRunner.kt
package com.mbmc.template.helper

import android.app.Application
import android.content.Context
import androidx.test.runner.AndroidJUnitRunner
import com.mbmc.template.TestApplication

class TestRunner: AndroidJUnitRunner() {
    override fun newApplication(cl: ClassLoader?, className: String?, context: Context?): Application
            = super.newApplication(cl, TestApplication::class.qualifiedName, context)
}